PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Data/Migrations/20241105115245_First.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace PrivateHospitals.Data.Migrations
```

Comment:
Remove ALL migrations and create 1 clean migration from scratch

Reply:
Fixed